web/themes/new_weather_theme/assets/sass/components/radar.scss

```diff
@@ -1,12 +1,57 @@
 @use "uswds-core" as *;
 
 .wx-radar-container {
-  aspect-ratio: 3/4;
+  aspect-ratio: 3/2;
+
+  &.wx-radar-container__expanded {
+    aspect-ratio: 3/4;
+  }
+
+  button.wx-radar-expand {
+    @include u-bg("transparent");
```

Comment:
I think this is actually not strictly necessary anymore. I added this when I was trying to get the button to be the same height as its contents, before I learned about the baseline vertical alignment magic margin. But I suppose it doesn't hurt anything, either?